V2/tezex/src/pages/Home/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { FC } from "react";
+
+import { NavHome } from "../../components/nav";
+import { Swap } from "../../components/swap";
+import { AddLiquidity } from "../../components/addLiquidity";
+import { RemoveLiquidity } from "../../components/removeLiquidity";
+
+import Grid2 from "@mui/material/Unstable_Grid2"; // Grid version 2
+type HomePaths = "swap" | "add" | "remove";
+
+export interface IHome {
+	path: HomePaths;
+}
+export const Home: FC<IHome> = (props) => {
```

Comment:
consider adding some sort of standard formatting like prettier

Reply:
I am using prettier but it formats to about 75 characters line length , what should I keep the line length to ? plus I can add it as a dev dependency  and auto format on build